mmengine/optim/scheduler/momentum_scheduler.py

```diff
@@ -289,7 +289,55 @@ class CosineRestartMomentum(MomentumSchedulerMixin,
 
 @PARAM_SCHEDULERS.register_module()
 class ReduceOnPlateauMomentum(ReduceOnPlateauParamScheduler):
```

Comment:
Why `ReduceOnPlateauMomentum` does not inherit `MomentumSchedulerMixin` here?

Reply:
According to mro, `ReduceOnPlateauMomentum` can't use step() method from `ReduceOnPlateauParamScheduler`.

Reply:
Great